pkgs/by-name/sd/SDL_gfx/package.nix

```
@@ -52,6 +52,7 @@ stdenv.mkDerivation (finalAttrs: {
     '';
     license = lib.licenses.zlib;
     teams = [ lib.teams.sdl ];
+    broken = stdenv.hostPlatform.isDarwin;
```

```suggestion
    badPlatforms = lib.platforms.darwin;
```
iirc marking badPlatforms gives nicer error output than broken, as broken has no way to report on which platforms it is actually broken